javascript-practice/src/js/controllers/product.detail.controller.js

```diff
@@ -0,0 +1,20 @@
+import findRoute from '../utils/findRoute';
+
+export default class ProductDetailController {
+  constructor(view, service) {
+    this.view = view;
+    this.service = service;
+
+    const { params } = findRoute(window.location.pathname);
```

Comment:
We have 2 chooses at here:
1. Move this line into displayProductById
2. Use this.productId = ...... Because we shouldn't define variable in constructor